### DISCUSSION TITLE
linter-pylint - no longer working

### DISCUSSION THREAD

**Discussion by kxkv** [pulsar-edit/.github#175] [Community Package Help] [answered]
the [linter-pylint](https://web.pulsar-edit.dev/packages/linter-pylint) package has stopped working with new versions of pylint.

are there any other python developers out there, that use pulsar and pylint? if so, how do you do it?

```
[Linter] Error running Pylint Error: pylint: Command line or configuration file:1: UserWarning: Specifying exception names in the overgeneral-exceptions option without module name is deprecated and support for it will be removed in pylint 3.0. Use fully qualified name (maybe 'builtins.Exception' ?) instead.
    at Object.lint (main.js:161)
    at async Promise.all (index 0)
    at Bt.lint (linter-registry.ts:171)
```

**Comment by kxkv** [comment, marked as answer]
after i updated the pylintrc file (regenerated it, using `pylint --generate-rcfile > .pylintrc`), pylint in pulsar is working well again. so, the error seems to have been caused by a discrepancy in between the rc file and the updated pylint.

**Comment by asiloisad** [comment]
you can check my package `linter-ruff`. a [ruff](https://beta.ruff.rs/docs/) is python linter which recode pylint (but not at all) and another linters
